@xen-orchestra/web/src/components/vm/VmGeneralInfo.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <UiPanel class="vm-general-info">
+    <UiTitle>
+      {{ $t('general-information') }}
+    </UiTitle>
+    <VtsQuickInfoRow :label="$t('name')">
+      <template #value>
+        <div class="text-ellipsis">
+          {{ vm.name_label }}
+        </div>
+      </template>
+    </VtsQuickInfoRow>
+    <VtsQuickInfoRow :label="$t('id')">
+      <template #value>
+        <div class="text-ellipsis">
+          {{ vm.name_label }}
+        </div>
+      </template>
+    </VtsQuickInfoRow>
+    <VtsQuickInfoRow :label="$t('name')">
+      <template #value>
+        <div class="text-ellipsis">
+          {{ vm.id }}
+        </div>
+      </template>
+    </VtsQuickInfoRow>
+    <VtsQuickInfoRow :label="$t('description')">
+      <template #value>
+        <div class="text-ellipsis">
+          {{ vm.name_description }}
+        </div>
+      </template>
+    </VtsQuickInfoRow>
+    <VtsQuickInfoRow :label="$t('tags')">
+      <template v-if="tags.length >= 1" #value>
+        <div class="tag-list">
+          <UiTagsList v-for="tag in tags" :key="tag">
+            <UiTag accent="info" variant="secondary">{{ tag }}</UiTag>
+          </UiTagsList>
+        </div>
+      </template>
+    </VtsQuickInfoRow>
+    <VtsQuickInfoRow :label="$t('os-name')">
+      <template #value>
+        <div v-if="vm.os_version.name" class="text-ellipsis">
+          {{ vm.os_version.name }}
+        </div>
+      </template>
+    </VtsQuickInfoRow>
+    <VtsQuickInfoRow :label="$t('os-kernel')">
+      <template #value>
+        <div v-if="vm.os_version.name" class="text-ellipsis">
+          {{ vm.os_version.distro }}
+        </div>
+      </template>
+    </VtsQuickInfoRow>
+    <VtsQuickInfoRow :label="$t('Management-agent-version')">
```

Comment:
```suggestion
    <VtsQuickInfoRow :label="$t('management-agent-version')">
```

Remember to change the other keys